src/libxrpl/protocol/STAmount.cpp

```diff
@@ -359,7 +278,7 @@ STAmount::xrp() const
 IOUAmount
 STAmount::iou() const
 {
-    if (mIsNative)
+    if (native() || holds<MPTIssue>())
```

Comment:
```suggestion
    if (native() || !holds<Issue>())
```
This removes the need to edit this line again if we add another issue type in the future.